tests/tools/cohort_tracking/test_cohort_tracking.py

```diff
@@ -0,0 +1,169 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import ehrapy as ep
+import ehrapy.tools.feature_ranking._rank_features_groups as _utils
+from ehrapy.io._read import read_csv
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_DATA_PATH = f"{CURRENT_DIR.parent}/test_data_features_ranking"
+
+
+def _compare_dict_equal(dict1, dict2, tolerance=1e-9):
+    if isinstance(dict1, dict) and isinstance(dict2, dict):
+        if set(dict1.keys()) != set(dict2.keys()):
+            return False
+        for key in dict1.keys():
+            if not _compare_dict_equal(dict1[key], dict2[key], tolerance):
+                return False
+        return True
+    elif isinstance(dict1, list) and isinstance(dict2, list):
+        if len(dict1) != len(dict2):
+            return False
+        for val1, val2 in zip(dict1, dict2):
+            if not _compare_dict_equal(val1, val2, tolerance):
+                return False
+        return True
+    elif isinstance(dict1, float) and isinstance(dict2, float):
+        return abs(dict1 - dict2) < tolerance
+    elif isinstance(dict1, str) and isinstance(dict2, str):
+        return dict1 == dict2
+    else:
+        return dict1 == dict2
+
+
+class TestCohortTracker:
+    @pytest.mark.parametrize("columns", [None, ["glucose", "weight", "disease", "station"]])
+    def test_CohortTracker_init_vanilla(self, columns):
+        adata = ep.io.read_csv(
```

Comment:
Maybe this should be a fixture?

Reply:
the creation of the adata? did that now, is better indeed